packages/catalog-search/src/data/constants.js

```diff
@@ -51,6 +51,8 @@ export const SEARCH_FACET_FILTERS = [
   },
 ];
 
+export const ADDITIONAL_FACET_FILTERS = ['skills.name', 'job_name'];
```

Comment:
Facet filter for job is now `name`

Reply:
Yes, you are right. But I've requested @saleem-latif to change it to job_name, as in URL it doesn't make sense what name is it.